BooleanKeypad/BooleanKeypad.cs

```diff
@@ -61,21 +87,25 @@ public class BooleanKeypad : MonoBehaviour {
 			for (int i = 0; i < 4; i++) {
 				if (this.buttonTruths[i] && !this.pressedButtons[i]) {
 					if (i < id) {
+						Debug.LogFormat("[Boolean Keypad #{0}] Button {1} pressed. This is not the next true button in reading order. Button {2} expected instead.", moduleId, id+1, i+1);
 						Strike();
 					}
+					Debug.LogFormat("[Boolean Keypad #{0}] Button {1} pressed. This is correct.", moduleId, id+1);
```

Comment:
I don't think this log is correct, because it happens when a button that was supposed to be pressed but doesn't get pressed. But it also happens when that buttons should've been pressed before this currently pressed button (resulting in a strike) and saying a button is correct after getting a strike is... dumb. I propose changing the code flow here and change the message.
```suggestion
					} else {
						Debug.LogFormat("[Boolean Keypad #{0}] Button {1} pressed. This is correct.", moduleId, id+1);
					}
```